samples/sid_end_device/include/sidewalk.h

```diff
@@ -53,7 +53,7 @@ int sidewalk_event_send(event_handler_t event, void *ctx, ctx_free free);
 #elif CONFIG_SIDEWALK_LINK_MASK_FSK
 #define DEFAULT_LM (uint32_t)(SID_LINK_TYPE_2)
 #elif CONFIG_SIDEWALK_LINK_MASK_LORA
-#define DEFAULT_LM (uint32_t)(SID_LINK_TYPE_1 | SID_LINK_TYPE_3)
+#define DEFAULT_LM (uint32_t)(SID_LINK_TYPE_3)
```

Comment:
You are not able to register when build with that link as default

Reply:
True, but this is default. Thus I can image scenario when FW is build with:
- CONFIG_SIDEWALK_LINK_MASK_LORA=y
- CONFIG_SID_END_DEVICE_PERSISTENT_LINK_MASK=n

FW boots, user switch to BLE, do registration, FW reset and we are in LORA